packages/mattermost-redux/src/types/general.ts

```diff
@@ -28,3 +28,15 @@ export enum LicenseSkus {
     Professional = 'professional',
     Enterprise = 'enterprise',
 }
+
+export const isEnterpriseLicense = (license?: ClientLicense) => {
```

Comment:
Yeah, this is tricky. The only more general term this could take is something along the lines of `High Tier` sku (Low Tier -> Starter, Mid Tier -> Professional/E10) but not sure that says much in developer context. 

On the other hand, reusing this by others without checking the code they may assume it's just the `Enterprise` sku but it's also allowing `E20`

Reply:
That's one potential misinterpretation, yes. In general, I think most folks have internalized `E20 ~= Enterprise` and `E10 ~= Professional`, but where it matters (i.e. a feature intended only for Cloud Enterprise), this attempt to group both might be confusing.